src/flowMC/resource/kernel/HMC.py

```
@@ -26,14 +26,14 @@ class HMC(ProposalBase):
     """
 
     condition_matrix: Float[Array, " n_dim"]
-    step_size: Float
+    step_size: float
     leapfrog_coefs: Float[Array, "n_leapfrog n_dim"]
```

Ruff (0.15.18)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)